modules/store/spec/selector.spec.ts

```diff
@@ -128,7 +128,7 @@ describe('Selectors', () => {
       const selectorFn = jasmine
         .createSpy(
           'selectorFn',
-          createSelector((state) => state, projectorFn)
+          createSelector((state: unknown) => state, projectorFn)
```

Comment:
the typing is slightly stricter now such that new errors like implicit any may occur. therefore, this is a breaking change.